source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/calification.js

```diff
@@ -517,6 +538,14 @@
         }
     }
 
+    EvaluationViewModel.prototype.toggleAllComments = function (data, event) {
+        var toggleAllTo = viewmodel.showCalificationsComments() === null ? false : !viewmodel.showCalificationsComments();
+        _.each(data.groups, function (group) {
+            _.each(group.items, function (item) {
+                item.showComments (toggleAllTo);
```

Comment:
I think that here it could be also clean to flattering the items, but in this case it is not so important.